pkg/reconciler/v1alpha1/trigger/resources/service.go

```diff
@@ -0,0 +1,52 @@
+/*
+Copyright 2019 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package resources
+
+import (
+	"fmt"
+
+	eventingv1alpha1 "github.com/knative/eventing/pkg/apis/eventing/v1alpha1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+// NewService returns a K8s placeholder service for trigger 't'.
+func NewService(t *eventingv1alpha1.Trigger) *corev1.Service {
```

Comment:
I prefer leaving this `NewK8sService` as we talk about both K8s and Knative Services frequently. I find it useful to distinguish between the two.

Reply:
I think we should use KnService and Service.

Reply:
> I think we should use KnService and Service.

thinking a bit here, I agree with this, instead of `newK8sService`